web/config/routes.ts

```diff
@@ -39,6 +39,10 @@ const routes = [
     path: '/routes/:rid/edit',
     component: './Route/Create',
   },
+  {
+    path: '/routes/:rid/duplicate',
```

Comment:
Do we need this path?

Reply:
> hi @batman-ezio
> pr #1606 has been merged. please sync codes from branch master.

sync from master but frontend still fails

Reply:
cc @guoqqqi to help resolve failed test

Reply:
Hi, I have fixed the problem with the relevant test and wait for #1608.

Reply:
@guoqqqi  I sync from master after your PR merged, but still failed